app/components/article-featured-video.js

```diff
@@ -23,16 +23,27 @@ export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
 
 	classNames: ['article-featured-video'],
 
+	// transparent gif
+	attributionAvatarUrl: 'data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==',
```

Comment:
we use the same gif in about 7 places
what do you think in moving it to
https://github.com/Wikia/mobile-wiki/blob/dev/app/utils/thumbnail.js ?